pkg/cmd/server/admin/create_keypair.go

```diff
@@ -112,7 +112,7 @@ func (o CreateKeyPairOptions) CreateKeyPair() error {
 		return err
 	}
 
-	fmt.Fprintf(o.Output, "Generated new key pair as %s and %s\n", o.PublicKeyFile, o.PrivateKeyFile)
```

Comment:
Yeah, need glog io.Writer wrapper

On Tue, Mar 15, 2016 at 2:16 PM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/server/admin/create_keypair.go
> <https://github.com/openshift/origin/pull/8024#discussion_r56213717>:
>
> > @@ -112,7 +112,7 @@ func (o CreateKeyPairOptions) CreateKeyPair() error {
> >  		return err
> >  	}
> >
> > -	fmt.Fprintf(o.Output, "Generated new key pair as %s and %s\n", o.PublicKeyFile, o.PrivateKeyFile)
>
> Yeah, this is the delegation thing I was talking about. @sosiouxme
> <https://github.com/sosiouxme> requested these as Printfs in the case
> where you're actually trying to run this command. When you run this command
> in a delegated case, you should pass an o.Output that actual writes to
> glog.V(3).Info.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8024/files/011a2f607bc9309896480a82044992be53dc3f3b#r56213717>
>